src/App.jsx

```diff
@@ -1,3 +1,24 @@
+import { useState } from "react";
+import "./App.css";
+import { Input } from "./components/Input";
+import { ThoughtRender } from "./components/ThoughtRender";
+import { ThoughtBox } from "./components/ThoughtBox";
+import { Header } from "./components/Header";
+
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
+  const [thoughts, setThoughts] = useState([]);
```

Comment:
aaah I see, you put your state variables here, I think that's what they recommended! I didn't.. thanks for the inspo 👯